runtime/vm/jvminitcommon.c

```diff
@@ -128,22 +128,31 @@ createLoadInfo(J9PortLibrary *portLibrary, J9Pool *aPool, const char *name, U_32
 
 /**
  * This method indicates if the requested jniVersion is valid or not
+ *
+ * Note: a JVMTI environment can be obtained through the JNI Invocation API GetEnv function
+ *       https://docs.oracle.com/en/java/javase/11/docs/specs/jvmti.html
  * @param jniVersion the version to be checked
  * @returns true if the version is valid
  */
 UDATA
 jniVersionIsValid(UDATA jniVersion)
 {
-	return (jniVersion == JNI_VERSION_1_1)
-		|| (jniVersion == JNI_VERSION_1_2)
-		|| (jniVersion == JNI_VERSION_1_4)
-		|| (jniVersion == JNI_VERSION_1_6)
-		|| (jniVersion == JNI_VERSION_1_8)
+	return (JNI_VERSION_1_1 == jniVersion)
+		|| (JNI_VERSION_1_2 == jniVersion)
+		|| (JNI_VERSION_1_4 == jniVersion)
+		|| (JNI_VERSION_1_6 == jniVersion)
+		|| (JNI_VERSION_1_8 == jniVersion)
 #if JAVA_SPEC_VERSION >= 9
-		|| (jniVersion == JNI_VERSION_9)
+		|| (JNI_VERSION_9 == jniVersion)
 #endif /* JAVA_SPEC_VERSION >= 9 */
 #if JAVA_SPEC_VERSION >= 10
-		|| (jniVersion == JNI_VERSION_10)
+		|| (JNI_VERSION_10 == jniVersion)
 #endif /* JAVA_SPEC_VERSION >= 10 */
+#if JAVA_SPEC_VERSION >= 11
+		|| (JVMTI_VERSION_1_0 == jniVersion)
```

Comment:
should this be `JVMTI_VERSION_11` ?

Reply:
Changed the condition to `J9_ARE_ALL_BITS_SET(jniVersion, JVMTI_VERSION_1)` to accept existing `JVMTI_VERSION_1x` versions.
JDK11 JVMTI doc [1] has an example `(*jvm)->GetEnv(jvm, &jvmti, JVMTI_VERSION_1_0)` which I am assuming later JVMTI versions should be accepted as well.

[1] https://docs.oracle.com/en/java/javase/11/docs/specs/jvmti.html